pkg/util/common.go

```diff
@@ -1552,20 +1551,14 @@ func GetKubernetesVersion(clientset *kubernetes.Clientset) (string, error) {
 	return "", err
 }
 
-// GetOcVersion will return the version of openshift
-func GetOcVersion(clientset *kubernetes.Clientset) (string, error) {
-	body, err := clientset.Discovery().RESTClient().Get().AbsPath("/version/openshift").Do().Raw()
+// IsOpenshift will return the version of openshift
+func IsOpenshift(clientset *kubernetes.Clientset) (bool, error) {
```

Comment:
is returning the error really useful here at all?  It doesn't look like it's really used at all.  If there's an error we assume we're not running on openshift and we don't abort under any condition or log the error message, so really only a bool is needed to be returned